recipes/ditaa/lein

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/env bash
```

Comment:
leinining is available on conda-forge, I think you should use it instead of copying these files here.